src/v/redpanda/admin/api-doc/config.json

```diff
@@ -26,6 +26,20 @@
     }
   }
 },
+"/v1/loggers": {
+  "get": {
+    "summary": "List all logger names",
+    "operationId": "get_loggers",
+    "produces": [
+      "application/json"
+    ],
+    "responses": {
+      "200": {
+        "description": "Loggers"
```

Comment:
Please write something meaningful here -- audience should be someone who is reading our admin API docs and might not know what kind of logger we mean.

Reply:
Done